src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -290,21 +312,23 @@ trait ClusterComponent extends LeoComponent {
     /* WARNING: The init bucket and SA key ID is secret to Leo, which means we don't unmarshal it.
      * This function should only be called at cluster creation time, when the init bucket doesn't exist.
      */
-    private def marshalCluster(cluster: Cluster, initBucket: String, serviceAccountKeyId: Option[ServiceAccountKeyId]): ClusterRecord = {
+    private def marshalCluster(cluster: Cluster,
+                               initBucket: String,
```

Comment:
I think `initBucket` will now need to be a `Option[String]` in `clusterQuery.save` and `marshalCluster`. In phase 2 we will be persisting clusters _before_ creating the init bucket, so will need to pass `None`.